docs/notebooks/sklearn_api.ipynb

```diff
@@ -56,17 +56,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
This isn't a good place for demonstration of this feature, can you create a distinct notebook with comparison (w/wo pivot norm)?

Reply:
Let me finish with the blog as both will need that comparison.